commands/docker-compose.ts

```diff
@@ -38,7 +38,7 @@ function computeItems(folder: vscode.WorkspaceFolder, uris: vscode.Uri[]): vscod
     return items;
 }
```

Comment:
![image](https://user-images.githubusercontent.com/6913354/53774616-16d30e80-3ea4-11e9-9ace-56f89597ce8e.png)

Reply:
@fiveisprime What do you think about the number of context menu items?

Reply:
My point was that you have 3 restarts.

Personally I would group up and down together in a group, and start/stop/restart in another group.

Reply:
We talked a little about this via email. I was also confused by the three restarts 🤣 My ¢2 on the order:

- Compose Up
- Compose Down
- Compose Restart
- Compose Stop
- Compose Start

Will `exec` come into this as well or are we less worried about that?

Reply:
Don't you think stop/stop should be in a separate group?  That would make it easier to find the one you want and also break up the related functionality (since start/stop are non-destructive and don't build).

Do you think we should consider keyboard bindings for some of these things?  Or best left to the user?

Reply:
Actually, I can't remember if restart does up/down or start/stop.

Reply:
It used to do up/down, but it should be doing stop/start (which I've fixed).

Reply:
> Don't you think stop/stop should be in a separate group?  That would make it easier to find the one you want and also break up the related functionality (since start/stop are non-destructive and don't build).
> Do you think we should consider keyboard bindings for some of these things?  Or best left to the user?

@fiveisprime I still think we should make groups.
![image](https://user-images.githubusercontent.com/6913354/55441552-19d32480-5560-11e9-8a85-1bd5a1249b82.png)